src/views/Graph.vue

```diff
@@ -216,18 +218,29 @@ export default {
       title: this.getPageTitle('App.workflow', { name: this.workflowName })
     }
   },
+  props: {
+    initialOptions: {
+      type: Object,
+      required: false,
+      default: () => {}
+    }
+  },
+  defaults: {
+    // the spacing between nodes
+    spacing: 1.5,
+    // if true layout is left-right is false it is top-bottom
```

Comment:
This took me a minute to figure out
```suggestion
    // if true, layout is left-right; if false, it is top-bottom
```